<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JiuGongGeProjectClass</class>
 <widget class="QMainWindow" name="JiuGongGeProjectClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JiuGongGeProject</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="ShowImageWidget" name="widget_ShowImage" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>360</width>
      <height>360</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_edit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>250</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>编辑排列</string>
    </property>
   </widget>
   <widget class="QComboBox" name="box_chooseAuto">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_chooseSize">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>210</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>选择排列大小M*N</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_begin">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>160</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_generate">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>生成</string>
    </property>
   </widget>
   <widget class="QComboBox" name="box_chooseMat">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="box_search">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_stop">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>160</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_end">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>160</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>终止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_beginSearch">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>120</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>开始搜索</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>时间</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="button_setTime">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>280</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>设置最大搜索时间</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_importImg">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>310</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>导入图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clearPic">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>310</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清除图片</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>722</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ShowImageWidget</class>
   <extends>QWidget</extends>
   <header>showimagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="JiuGongGeProject.qrc"/>
 </resources>
 <connections/>
</ui>
